test/unit/test_pressure_reconstruction.py

```diff
@@ -0,0 +1,760 @@
+import numpy as np
```

Comment:
There is a test file, uni/test_fv_boundary_pressure_reconstrution or similar, which already contains test. Merging that file with this one would be good.